index.html

```diff
@@ -17,18 +17,29 @@
   <title>Microverse Portifolio - Template 3</title>
 </head>
 <body>
+  <section id="mobile-expanded-menu" class="mobile-expanded-menu disabled">
+    <nav id="expanded-nav" class="expanded-nav flex-container-column">
+      <a class="close-btn" href="#"><img id="close-btn" class="close-img" src="./css/img/mobile_menu/Vector.svg" alt=""></a>
+      
+      <ul id="expanded-menu-list" class="expanded-menu-list">
```

Comment:
- [ ] Your fonts used for the mobile menu does not match the font in the provided template. Kindly ensure to stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468)

**Your Mobile menu design:**
![image](https://user-images.githubusercontent.com/12745474/127226470-4d260414-3a1d-493d-90c2-910ad6ba542a.png)


**The original Mobile menu design:**
![image](https://user-images.githubusercontent.com/12745474/127226753-5f109add-3570-4fe3-82bd-d704ab6ad0f4.png)